python/dlisio/__init__.py

```diff
@@ -12,6 +12,89 @@
 except pkg_resources.DistributionNotFound:
     pass
 
+def get_encodings():
+    """Get codepages to use for decoding strings
+
+    Get the currently set codepages used when decoding strings.
+
+    Returns
+    -------
+    encodings : list
+
+    See also
+    --------
+    set_encodings
+    """
+    return core.get_encodings()
+
```

Comment:
Maybe fingerprints should just be byte strings anyway?

Reply:
That's internal, so it doesn't matter what it would be. It might be byte string or we can just use the encoding that simply considers any sequence of characters valid (as I think ISO-8859-1 might)

Reply:
Well, bytestrings have few round-trip problems, and I guess is conceptually simpler and less demanding than dealing with str. Guess it's a task, then?

Reply:
Sure, I'll create an issue out of it (and test files) along with all other issues I saw.

Just one thing about byte strings. If user would ask for a frame with name "COOL-°-FRAME" or with any Chinese/French characters in it, user might not get it. But I am not sure if that's possible that any frame user is actually interested in has this weird name.

Reply:
Not through the fingerprint anyway, as that is ours to play with.